protocol/deputy-pause-module.md

```diff
@@ -0,0 +1,80 @@
+# Deputy Pause Module
+
+## Context
+
+The Foundation Safe is given the ability to act as the Security Council Safe for a limited number
+of safety-net actions on OP Mainnet. For instance, the Foundation Safe can act as the Security
+Council Safe to trigger the Superchain-wide pause function. These powers are given to the
+Foundation Safe through the `DeputyGuardianModule` installed in the Security Council Safe.
+
+## Problem Statement
+
+The Foundation Safe pre-signs certain transactions to reduce response time in case of an emergency.
+Because pre-signed Safe transactions are dependent on the nonces of that Safe, these pre-signed
```

Comment:
```suggestion
Because pre-signed Foundation Safe transactions are dependent on the nonces of that Foundation Safe, these pre-signed
```

Reply:
Disambiguating which safe we refer to, as there are two safes.

Reply:
There's also two foundation safes :) 
```suggestion
Because pre-signed Foundation Operations Safe transactions are dependent on the nonces of the Foundation Operations Safe, these pre-signed
```

Reply:
Ah this is mentioned in the next paragraph, I'm indifferent to how this thread is resolved